### DISCUSSION TITLE
Deployments do not indicate who triggered them

### DISCUSSION THREAD

**Discussion by tvd0x2a** [craftcms/cloud#25] [Feature Requests & Enhancements]
Manually triggered deployments do not show who triggered them. In some cases, it seems like Pixel & Tonic staff may, understandably/reasonably, be triggering them (or maybe an automated system?).

It'd be great to be able to see who is triggering the manual deployments!

**Comment by jasonmccallister** [comment]
@tvd0x2a we have had this discussion a few times, I agree it is something we should expose in Console. Thank you for the feedback!
